src/main/java/org/zaproxy/zap/extension/jwt/JWTExtension.java

```diff
@@ -32,24 +49,77 @@
 
     protected static final Logger LOGGER = Logger.getLogger(JWTExtension.class);
 
+    private static final List<Class<? extends Extension>> DEPENDENCIES;
+
+    private HttpFuzzerHandler httpFuzzerHandler;
+    private JWTMessageLocationReplacerFactory jwtMessageLocationReplacerFactory;
```

Comment:
Maybe move these declarations out from between the dependencies constant and their static declaration.

Reply:
hmm sure.